Library/Formula/bash-completion.rb

```diff
@@ -32,8 +33,15 @@ def install
 
     system "./configure", "--prefix=#{prefix}"
     system "make", "install"
+  end
 
-    unless (compdir/"brew_bash_completion.sh").exist?
+  def post_install
+    # This symlink needs to be created in the post-install step. Otherwise, a
+    # copy of our official bash completion for `brew` will end up in the bottle,
+    # which is not very helpful in keeping this up-to-date as Homebrew updates.
+    brew_comp = compdir/"brew_bash_completion.sh"
+    unless brew_comp.symlink?
```

Comment:
Then presumably this code is going to go away very soon since I doubt you want to be unconditionally blowing away a file with a magic name in etc forever?